builddefs/bootloader.mk

```diff
@@ -190,8 +190,8 @@ ifeq ($(strip $(BOOTLOADER)), stm32duino)
     DFU_ARGS = -d 1EAF:0003 -a 2 -R
     DFU_SUFFIX_ARGS = -v 1EAF -p 0003
 endif
-ifeq ($(strip $(BOOTLOADER)), tinyuf2)
-    OPT_DEFS += -DBOOTLOADER_TINYUF2
+ifeq ($(strip $(BOOTLOADER)), uf2-tinyuf2)
```

Comment:
I'm not sure about this change anymore. Maybe it would be better to keep it as tinyuf2 and name the upcoming bootloader support for rp2040 mcus just `rp2040` instead of `uf2-rp2040`? Reasoning here is that they are both uf2 compatible but actually different bootloaders and for the end-user this is probably not important anyways because it is an implementation detail. (In the end it is just a matter of taste I guess.)

Reply:
tbh tinyuf2 is an indicator that it has a specific entry procedure from software. RP2040 has its own as well, so realistically I'd be inclined to keep it at `tinyuf2` and `rp2040`. Was more swapped to these to match your fork's RP2040 impl.

tinyuf2 has different requirements for entry for different MCUs anyway, so it'd make sense to keep them all aligned that way - as far as I'm concerned marking things as "uf2-xxxx" doesn't tell much because it's the firmware format, not the bootloader itself.

Reply:
Full ACK, I'll revert the naming in the rp2040 PR!